lib/themes/dosomething/paraneue_dosomething/template.php

```diff
@@ -92,6 +94,7 @@ function paraneue_dosomething_js_alter(&$js) {
 
   // Force removal of files that we can't seem to kill with exclude[js] array
   unset($js['profiles/dosomething/modules/contrib/devel/devel_krumo_path.js']);
+  unset($js['profiles/dosomething/modules/contrib/google_analytics/googleanalytics.js']);
```

Comment:
Not removing the module, just removing some extra Drupal-specific JS that I don't believe we're using (such as custom rules for some lightbox plugin and CTools things). It still adds the Analytics snippet based on the `UA-XXXXXX-XX` site ID in the module settings.